core/indigo-core/molecule/src/sequence_loader.cpp

```diff
@@ -386,8 +386,14 @@ void SequenceLoader::addMonomer(KetDocument& document, const std::string& monome
         }
 
         std::vector<KetAmbiguousMonomerOption> options;
+        const std::string uracil_alias = "U";
+        const std::string thymine_alias = "T";
+
         for (auto template_alias : alternatives.value().get())
         {
+            // filter out uracil and thymine for DNA and RNA respectively
```

Comment:
Maybe just change T to U for RNA? Like
`if (seq_type == SeqTypeRNASeq && template_alias=='T')
  template_alias='U'`

Reply:
fixed